homeassistant/components/camera/onvif.py

```diff
@@ -72,7 +72,8 @@
 })
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax. Please drop this decorator and prefix async methods with `async` like `async def async_setup_platform`. Replace any `yield from` with `await`.